frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingOption/EmbeddingSdkOptionCard/tests/setup.tsx

```diff
@@ -64,16 +67,12 @@ export function setup({
 
   setupSettingsEndpoints(settingDefinitions);
   setupPropertiesEndpoints(settingValues);
+  setupUpdateSettingEndpoint();
+  setupUpdateSettingsEndpoint();
 
-  const updateSetting = jest.fn();
-  renderWithProviders(
-    <EmbeddingSdkOptionCard updateSetting={updateSetting} />,
-    {
-      storeInitialState: state,
-    },
-  );
-
-  return {
-    updateSetting,
-  };
+  renderWithProviders(<EmbeddingSdkOptionCard />, { storeInitialState: state });
+  await waitFor(async () => {
+    const gets = await findRequests("GET");
+    expect(gets).toHaveLength(2);
```

Comment:
Might be nice to add an inline comment on what kind of requests we are expecting, as I'm not sure what the requests are. (My guess would be instance properties and user settings)